README.md

```diff
@@ -1,70 +1,220 @@
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.dev/">ReactJS</a></li>
+    <li><a href="#">CSS</a></li>
+  </ul>
+</details>
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+<!-- Features -->
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+### Key Features <a name="key-features"></a>
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+- **Use Functional Components**
+- **Implement lifecycle hooks**
+- **Use store, actions and reducers in React**
+- **Implement Routing using React Router**
+- **Style using CSS modules**
+- **Write unit tests with React Testing Library and Jest**
 
-## Learn More
+<!-- > Describe between 1-3 key features of the application. -->
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+<!-- LIVE DEMO -->
 
-### Code Splitting
+## 🚀 Live Demo <a name="live-demo"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+<!-- > Add a link to your deployed project. -->
 
-### Analyzing the Bundle Size
+- <a href="https://649fea9d8cd57a0c58cb1cbc--balancesheet-biftu.netlify.app/">Space Travelers' Hub Live</a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Making a Progressive Web App
+<!-- Video -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+## 🎥 Video <a name="video"></a>
 
-### Advanced Configuration
+<!-- > Add a link to your video project. -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+<a href="https://www.loom.com/share/15131936398e4c0397663968e4bc0e8c?sid=3244ce95-7795-46cf-aea9-afdbd3e0828c">Get an understanding of the project by watching this video</a>
+
```

Comment:
- [ ] In your video presentation, the last requirement was for your to highlight a piece of code you're proud of or a feature you're most proud of. I noticed you did talk about the search bar and also how you optimized the API with 1000s of values, but didn't show how it was implemented in your code. Kindly remake the video and highlight that section 👍 

### Requirements:
![image](https://github.com/Bifabig/react-redux-capstone/assets/50990925/2f614756-b715-416d-96a9-4780009842b5)